mmaction/models/backbones/mobilenet_v2.py

```diff
@@ -129,8 +129,9 @@ class MobileNetV2(nn.Module):
             channels in each layer by this amount. Default: 1.0.
         out_indices (None or Sequence[int]): Output from which stages.
             Default: (7, ).
-        frozen_stages (int): Stages to be frozen (all param fixed).
-            Default: -1, which means not freezing any parameters.
+        frozen_stages (int): Stages to be frozen (all param fixed). Note that
+            the last stage in ``MobileNetV2`` is ``conv2``. Default: -1,
+            which means not freezing any parameters.
```

Comment:
The applicable options of frozen_stages should be range(0, 9) now, please modify the assertion.

Reply:
It is still `range(-1, 8)`.